### PR TITLE
Revert "Revert "Humid Heat Metrics""

### PR DESCRIPTION
**Pull request by jukent**
Reverts NCAR/geocat-applications#116

New PR for HHM

### CODE REVIEW

**Review by anissa111 — APPROVED** [review]

File: applications/data_analysis/humid_heat_metrics.ipynb [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+    "\n",
+    "However, this formula is complicated by the reality that Natural Wet Bulb Temperature and Globe Temperature are not always readily available variables from weather stations or atmospheric models.\n",
+    "\n",
+    "In this notebook, we will demonstrate the Australian Bureau of Meteorology (ABM) and Bernard methods of predicting wet bulb global temperature with a focus on the July 1995 Chicago heatwave.\n",
+    "\n",
+    "For our analysis, we have ERA5 reanalysis data for the lower contiguous United States (50&deg;N, 24&deg;S, -66&deg;E, -125&deg;W) from July 1995 with the variables: 2-meter temperature, 2-meter dew point temperature, surface pressure, and *u*/*v* wind components.\n",
+    "\n",
+    "ERA5 is a reanalysis global weather/climate dataset that combines model output and ovservationss with physical understanding to create a spatially and temporally consistent historic dataset, spanning from 1940 to today (updated every 5 days){footcite}`era5`."
```
Comment (kafitzgerald) [inline applications/data_analysis/humid_heat_metrics.ipynb:32]:
```suggestion
    "ERA5 is a reanalysis, a global weather/climate dataset that combines model output and observations with physical understanding to create a spatially and temporally consistent historic dataset, spanning from 1940 to today (updated every 5 days){footcite}`era5`."
```

File: applications/data_analysis/humid_heat_metrics.ipynb [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+    "lon_chicago = -87.6298"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Australian Bureau of Meteorology (ABM)\n",
+    "\n",
+    "The [Australian Bureau of Meteorology](http://www.bom.gov.au/)'s method of estimating Wet Bulb Global Temperature is attractive due to its simplicity. It only requires temperature and water pressure (or relative humidity){footcite}`abm_2010`.\n",
```
Comment (kafitzgerald) [inline applications/data_analysis/humid_heat_metrics.ipynb:95]: water pressure?

File: applications/data_analysis/humid_heat_metrics.ipynb [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+    "    )  # water vapor pressure [hPa]\n",
+    "    wbgt = (0.567 * t_a) + (0.393 * p) + 3.94\n",
+    "    return wbgt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use our ERA5 data in this equation, we need to first calculate relative humidity (the ratio of vapor pressure to saturation pressure) from temperature and dewpoint. To do this we use the Magnus-Tetens Approximation for vapor pressures{footcite}`monteith_2008`:\n",
```
Comment (kafitzgerald) [inline applications/data_analysis/humid_heat_metrics.ipynb:122]:
```suggestion
    "To use our ERA5 data in this equation, we need to first calculate relative humidity (the ratio of vapor pressure to saturation pressure) from temperature and dewpoint. To do this we use the Magnus-Tetens Approximation for vapor pressure{footcite}`monteith_2008`:\n",
```

**Review by kafitzgerald — COMMENTED** [review]
Overall this looks good.  

Caught a few typos and formatting issues though (see comments).

File: applications/data_analysis/humid_heat_metrics.ipynb (hunk 2 above)
Comment (jukent) [inline applications/data_analysis/humid_heat_metrics.ipynb:95]:
```suggestion
    "The [Australian Bureau of Meteorology](http://www.bom.gov.au/)'s method of estimating Wet Bulb Global Temperature is attractive due to its simplicity. It only requires temperature and water vapor pressure (or relative humidity){footcite}`abm_2010`.\n",
```

File: applications/data_analysis/humid_heat_metrics.ipynb [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "Humid heat metrics index temperature and humidity, which is often more useful than comparing either variable alone when considering perceived temperature, as they together affect the human body's ability to cool itself down. Humid heat metrics are especially important for the safety of outside workers, the elderly, or otherwise high-risk, high-exposure individuals{footcite}`li_2020`.\n",
+    "\n",
+    "Wet Bulb Globe Temperature ({math}`WBGT`) is a measure of heat stress. The equation for outdoor ({math}`WBGT_{od}`) and indoor/shaded ({math}`WBGT_{id}`) WBGTs are:\n",
```
Comment (jukent) [inline applications/data_analysis/humid_heat_metrics.ipynb]:
```suggestion
    "Wet Bulb Globe Temperature ({math}`WBGT`) is a measure of heat stress. The equation for outdoor ({math}`WBGT_{od}`) and indoor/shaded ({math}`WBGT_{id}`) WBGTs are{footcite}`li_2020`:\n",
```

Comment (jukent) [inline applications/data_analysis/humid_heat_metrics.ipynb]:
```suggestion
    "Wet Bulb Globe Temperature ({math}`WBGT`) is a measure of heat stress. The equation for outdoor ({math}`WBGT_{od}`) and indoor/shaded ({math}`WBGT_{id}`) WBGTs are{footcite}`li_2020`:\n",
```

File: applications/data_analysis/humid_heat_metrics.ipynb [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+    "\n",
+    "{math}`WBGT_{od} = 0.7*T_{nwb} + 0.2*T_g + 0.1*T_a`\n",
+    "\n",
+    "{math}`WBGT_{id} = 0.7*T_{nwb} + 0.3*T_g`\n",
+    "\n",
+    "where {math}`T_a` refers to Dry Bulb Ambient Temperatue, {math}`T_{nwb}` is the Natural Wet Bulb Temperature with exposure to wind and sun, and {math}`T_g` is the Globe Temperature taken from inside a copper globe painted black and exposed to the sun{footcite}`li_2020`.\n",
```
[the remark this answers is not in the document]
Comment (jukent) [reply applications/data_analysis/humid_heat_metrics.ipynb:24]: I added a citation before the equations, does that work?

File: applications/data_analysis/humid_heat_metrics.ipynb [hunk 6]
```diff
@@ -0,0 +1,406 @@
...
+    "ax.plot(lon_chicago, lat_chicago, 'k*');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Additional Reading\n",
+    "\n",
+    "- [Legg, Robert. “Properties of Humid Air.” Air Conditioning System Design, Butterworth-Heinemann, Elsevier, pp. 1–28, doi:10.1016/B978-0-08-101123-2.00001-7.](https://www.sciencedirect.com/science/article/abs/pii/B9780081011232000017?via%3Dihub)"
```
[the remark this answers is not in the document]
Comment (jukent) [reply applications/data_analysis/humid_heat_metrics.ipynb]: Removed it

File: applications/data_analysis/humid_heat_metrics.ipynb (hunk 2 above)
Comment (kafitzgerald) [inline applications/data_analysis/humid_heat_metrics.ipynb]:
```suggestion
    "The [Australian Bureau of Meteorology](http://www.bom.gov.au/)'s method of estimating Wet Bulb Global Temperature is attractive due to its simplicity. It only requires temperature and relative humidity {footcite}`abm_2010`.\n",
```

Water pressure is something a little different (esp. in hydrology / hydraulics).  I think just leaving it at RH works.

Comment (jukent) [reply applications/data_analysis/humid_heat_metrics.ipynb]: Do you know why it says "this suggestion is out of date and cannot be applied? I'll do a work around, but I would like to just commit your change.

Comment (jukent) [reply applications/data_analysis/humid_heat_metrics.ipynb]: I thought I changed it to "vapor pressure" that's probably why it is out of sync?

Comment (kafitzgerald) [reply applications/data_analysis/humid_heat_metrics.ipynb]: Oh, maybe I found an older version?  Regardless, I'll go ahead and resolve these.

**Pull request merged**